internal/actioninfo/actioninfo.go

```diff
@@ -1,9 +1,29 @@
 package actioninfo
 
+import (
+	"fmt"
+	"log"
+)
+
 type DataParser interface {
-	// TODO: добавить методы
+	Parse(datastring string) error
+	ActionInfo() (string, error)
 }
 
 func Info(dataset []string, dp DataParser) {
-	// TODO: реализовать функцию
+	for index, data := range dataset {
+		err := dp.Parse(data)
+
+		if err != nil {
+			log.Printf("parsing error index: %d\n", index)
```

Comment:
Теряешь сообщение с ошибкой
```suggestion
			log.Printf("parsing error index %d: %v\n", index, err)
```